TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/DriveVelocityPIDTuner.java

```diff
@@ -44,20 +41,37 @@
  *    mitigate oscillations.
  * 2. Add kI (or adjust kF) until the steady state/constant velocity plateaus are reached.
  * 3. Back off kP and kD a little until the response is less oscillatory (but without lag).
+ *
+ * Pressing X (on the Xbox and Logitech F310 gamepads, square on the PS4 Dualshock gamepad) will
+ * pause the tuning process and enter driver override, allowing the user to reset the position of
+ * the bot in the event that it drifts off the path.
+ * Pressing A (on the Xbox and Logitech F310 gamepads, X on the PS4 Dualshock gamepad) will cede
+ * control back to the tuning process.
  */
 @Config
 @Autonomous(group = "drive")
 public class DriveVelocityPIDTuner extends LinearOpMode {
     public static double DISTANCE = 72; // in
 
-    private static final String PID_VAR_NAME = "VELO_PID";
+    public static double VX_WEIGHT = 1;
+    public static double VY_WEIGHT = 1;
+    public static double OMEGA_WEIGHT = 1;
```

Comment:
Perhaps this weighting logic should move to the drive class. Something like `setWeightedDrivePower()`.

Reply:
To `SampleMecanumDrive` or the `MecanumDrive`/`Drive` class (considering that `setDrivePower` is in `MecanumDrive`)?

Reply:
`SampleMecanumDrive` and `SampleTankDrive`